tests/balloonpanel/balloonpanelview.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* global document, Event */
+/* bender-tags: ui, balloonPanel, browser-only */
+
+import BalloonPanel from '/ckeditor5/ui/balloonpanel/balloonpanel.js';
+import BalloonPanelView from '/ckeditor5/ui/balloonpanel/balloonpanelview.js';
+import Model from '/ckeditor5/ui/model.js';
+
+describe( 'BalloonPanelView', () => {
+	let model, view;
+
+	beforeEach( () => {
+		model = new Model( {
+			maxWidth: 200
+		} );
+
+		view = new BalloonPanelView();
+
+		return new BalloonPanel( model, view ).init();
```

Comment:
I don't think we need `BalloonPanel` constructor here at all. It is not used anywhere in tests, isn't it? And besides, we should be more like in this ticket from now on https://github.com/ckeditor/ckeditor5-ui-default/issues/26, which is testing controllers and views separately.